docs/MANUAL.md

```diff
@@ -137,7 +137,7 @@ To start the server, type `./ergo run` and hit enter, and the server should be r
 
 1. Pull the latest version of Ergo: `docker pull oragono/oragono:latest`
 1. Create a volume for persistent data: `docker volume create oragono-data`
-1. Run the container, exposing the default ports: `docker run -d --name oragono -v oragono-data:/ircd-data -p 6667:6667 -p 6697:6697 oragono/oragono:latest`
+1. Run the container, exposing the default ports: `docker run -d --name oragono -v oragono-data:/ircd-data -p 6667:6667 -p 6697:6697 ghcr.io/ergochat/ergo:stable`
```

Comment:
Might as well update the rest of the example to say `ergo` instead of `oragono`, no? :)

Reply:
Sure. I've gone ahead and done that everything I felt confident to I did not update the CrowdIn link, as it appears the reference to the oragono project name is still valid (or at least used in other areas of the source).